README.md

```diff
@@ -17,3 +19,30 @@
 ## Getting Started
 
 Azure のリソースのデプロイおよびプロビジョニングについては、 [src/arm-template/README](src/arm-template/README.md) をご参照ください。
+
+</details>
+
+<details>
+  <summary>ENGLISH</summary>
+
+# Smart Store sample application
+
+This repository is a sample implementation based on [Smart Store Reference Architecture] (https://news.microsoft.com/en-us/2019/01/29/blog-smart-store/).
+
+: warning: _This document supports in Japanese only for now, sorry._
```

Comment:
Remove this line because the warning is for non-Japanese.

Reply:
Done

Reply:
My feedback is that remove the warning from the English part. Because the warning is for non-japanese.  But You removed a sentence at the Japanese part. It is different from my feedback. Please do carefully.